src/core/Akka.TestKit.Tests/TestKitBaseTests/DilatedTests.cs

```diff
@@ -6,17 +6,19 @@
 //-----------------------------------------------------------------------
 
 using System;
+using System.Diagnostics;
 using Akka.TestKit;
 using Xunit;
+using Xunit.Sdk;
 
 namespace Akka.Testkit.Tests.TestKitBaseTests
 {
     public class DilatedTests : AkkaSpec
     {
         private const int TimeFactor = 4;
-        private const int Timeout = 50;
+        private const int Timeout = 1000;
```

Comment:
Need a value big enough that other timing noise (such as GC) can be excluded from the result